### PR TITLE
UF-10071: Updated template readme

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,160 @@
 # TemplateSpringBoot
 
+_A concise description of what this Spring Boot microservice does._
+
 ## Status
```
Comment (foobar-1337) [inline README.md:5]: Put this at the bottom (after "License") and rename to "Code status"?

**Comment by foobar-1337** [comment]
image

File: README.md [hunk 2]
```diff
@@ -1,12 +1,160 @@
...
+
+  Use Spring profiles (`dev`, `prod`, etc.) to manage different configurations for different environments.
+
+- **Logging Configuration:**
+
+  Adjust logging levels if necessary.
+
+## Contributing
+
+Contributions are welcome! Please see [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines.
```
Comment (foobar-1337) [inline README.md:143]: Reference the file in the .github-repo?

Comment (cheezi0747) [reply README.md:143]: It will autoresolve when the PR is merged.

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**